### DISCUSSION TITLE
困扰一天了，pnpm创建的umi4项目，antd pro模板，什么都没有改就直接报错！！！！

### DISCUSSION THREAD

**Discussion by Cooper1886** [umijs/umi#11131] [Q&A] [answered]
最新创建的项目，但是各种引用报错，试了moduleResolution：node，试了改paths各种还是不行， 但是运行又可以运行！！！
![image](https://github.com/umijs/umi/assets/107901138/e1411581-65ee-455f-bb32-c9dd74f6f289)

**Comment by fz6m** [comment, marked as answer]
需要升级 vscode 版本，或安装 TS nightly 插件，或配置使用当前工作区项目内的 TS 作为引擎。

**Reply by Cooper1886** [reply]
感谢
